src/test/java/com/klarna/hiverunner/builder/HiveShellBaseTest.java

```diff
@@ -52,259 +52,259 @@
 @RunWith(MockitoJUnitRunner.class)
 public class HiveShellBaseTest {
 
-    @Rule
-    public TemporaryFolder tempFolder = new TemporaryFolder();
+  @Rule
```

Comment:
Can you put the indentations here back to how they were? Otherwise it's hard to spot what actually changed.